src/routes/SelectRepos/index.js

```diff
@@ -82,6 +82,20 @@ export class SelectRepos extends React.Component {
           Can&#8217;t find one of your repos here? Make sure you are watching it
           on Github!
         </p>
+        {this.props.selectedRepos.length ? (
+          <p className={theme.intro}>
```

Comment:
the content here is not a paragraph so shouldn't be wrapped in a `<p>`

Reply:
Should this be a `<span>` or just the button?

Reply:
I imagine it will need to go into a div for styling/positioning